app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue

```diff
@@ -135,8 +136,6 @@ $label-badge-size: var(--space-slab);
 
 .menu-label {
   flex-grow: 1;
-  display: inline-flex;
-  align-items: center;
```

Comment:
Will fix this overflow hidden issue.
<img width="191" alt="Screenshot 2022-12-05 at 10 39 04 PM" src="https://user-images.githubusercontent.com/64252451/205699609-3b181405-4a85-4b92-90ce-64939dbf4181.png">